src/components/backLink/stories.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+// import iconPointer from "../../../images/icon-pointer.png";
```

Comment:
remove commented out code, also avoid import from png if you can e.g. make svg components